Find square root of Given number. Handle invalid input through exception handling

Input should not be less than zero

1. squareRoot() Function is defined which returns -1 if input to the function is less than zero otherwise, it will return calculated value.
2. Test cases are written in test_cases.cpp using assert() function.
3. In main, inside try block, if function returns -1 (i.e. input is less than zero), then num is thrown and it is catched by catch(double num) block and proper message is displayed.